ci-operator/config/shiftstack/prune/shiftstack-prune-main.yaml

```diff
@@ -25,7 +25,7 @@ images:
 resources:
   '*':
     limits:
-      memory: 1Gi
+      memory: 4Gi
```

Comment:
4 might be too much. Have you tried with 2?

Reply:
No. Let's try with 2Gi first.

Reply:
Done